docs/applications/simple-bank-account/src/main/java/com/scalar/application/bankaccount/service/AccountService.java

```diff
@@ -68,21 +68,21 @@
 
   private ResponseEntity<String> serve(ThrowableFunction f, JsonObject json) {
     try {
-      ContractExecutionResponse response = f.apply(json);
-
-      if (response.getStatus() == StatusCode.OK.get()) {
-        return ResponseEntity.ok(response.getResult());
-      } else {
-        return ResponseEntity.badRequest()
-            .body("status: " + response.getStatus() + ", message: " + response.getMessage());
-      }
+      ContractExecutionResult result = f.apply(json);
+
+      return ResponseEntity.ok(result.getResult().toString());
+    } catch (ClientException e) {
+      return ResponseEntity.badRequest()
```

Comment:
It is not you, but I don't think it's good to return `BAD REQUEST(400)` here.
Can we return 500 instead (, which is also not correct, but better than 400) ?

Reply:
Thank you for review comment.
Changes updated in d02b87a